llm_on_ray/finetune/finetune.py

```diff
@@ -271,13 +273,20 @@ def train_func(config: Dict[str, Any]):
     elif device in ["hpu"]:
         from optimum.habana.transformers import GaudiTrainer
         from optimum.habana.transformers import GaudiTrainingArguments
+        from optimum.habana import GaudiConfig
 
+        gaudi_config = GaudiConfig()
+        gaudi_config.use_fused_adam = True
```

Comment:
Maybe should add a option in yaml config to use fused_adam on HPU fine-tuning.
fused_adam is provided in optimum-habana, this is only for HPU fine-tuning.